src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__top {
+    display: flex;
+    justify-content: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__top-img {
+    width: 100%;
+  }
+
+  &__bottom {
+    margin-top: 40px;
+
+    &-title {
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      color: $mainAccent;
```

Comment:
The variable `$mainAccent` is used for the color property. Ensure that this variable is defined in your SCSS files before using it. Otherwise, it will cause a compilation error.